graphql_api/types/repository/repository.graphql

```diff
@@ -1,15 +1,26 @@
+# Repository is a named collection of files uploaded
```

Comment:
btw if you follow this notation:
https://spec.graphql.org/June2018/#sec-Descriptions
these doc strings would appear in the ariadne gql playground tool.